package.json

```diff
@@ -27,12 +27,15 @@
   "dependencies": {
     "async": "2.1.4",
     "emits": "3.0.0",
+    "got": "7.1.0",
     "htmlparser2": "3.9.2",
     "image-size": "0.5.1",
+    "lodash": "4.17.4",
```

Comment:
In sub dependencies/libraries we should always use the caret.

Reply:
See one line below, the lib required single lodash functions. Either we continue or you can remove the extra line.